fluxcd/metadata.csv

```diff
@@ -1,13 +1,27 @@
-fluxcd.gotk.reconcile.duration.seconds.sum,count,,second,,Sum of the duration in seconds of a GitOps Toolkit resource reconciliation.,0,fluxcd,reconcile duration seconds,
-fluxcd.gotk.reconcile.duration.seconds.count,count,,second,,Count of the duration in seconds of a GitOps Toolkit resource reconciliation.,0,fluxcd,reconcile duration seconds,
-fluxcd.gotk.reconcile.duration.seconds.bucket,count,,second,,Bucket of the duration in seconds of a GitOps Toolkit resource reconciliation.,0,fluxcd,reconcile duration seconds,
 fluxcd.controller.runtime.active.workers,gauge,,worker,,Number of currently used workers per controller.,0,fluxcd,controller runtime active workers,
-fluxcd.controller.runtime.reconcile.count,count,,,, Total number of reconciliations per controller.,0,fluxcd,controller runtime reconcile total,
-fluxcd.controller.runtime.reconcile.time.seconds.bucket,count,,second,,Bucket of length of time per reconciliation per controller.,0,fluxcd,controller runtime reconcile time seconds,
-fluxcd.controller.runtime.reconcile.time.seconds.sum,count,,second,,Sum of length of time per reconciliation per controller.,0,fluxcd,controller runtime reconcile time seconds,
-fluxcd.controller.runtime.reconcile.time.seconds.count,count,,second,,Count of length of time per reconciliation per controller.,0,fluxcd,controller runtime reconcile time seconds,
 fluxcd.controller.runtime.max.concurrent.reconciles,gauge,,,,Maximum number of concurrent reconciles per controller.,0,fluxcd,controller runtime max concurrent reconciles,
+fluxcd.controller.runtime.reconcile.count,count,,,,Total number of reconciliations per controller.,0,fluxcd,controller runtime reconcile total,
 fluxcd.controller.runtime.reconcile.errors.count,count,,error,,Total number of reconciliation errors per controller.,0,fluxcd,controller runtime reconcile errors total,
+fluxcd.controller.runtime.reconcile.time.seconds.bucket,count,,second,,Bucket of length of time per reconciliation per controller.,0,fluxcd,controller runtime reconcile time seconds,
+fluxcd.controller.runtime.reconcile.time.seconds.count,count,,second,,Count of length of time per reconciliation per controller.,0,fluxcd,controller runtime reconcile time seconds,
+fluxcd.controller.runtime.reconcile.time.seconds.sum,count,,second,,Sum of length of time per reconciliation per controller.,0,fluxcd,controller runtime reconcile time seconds,
+fluxcd.gotk.reconcile.condition,gauge,,,,The current condition status of a GitOps Toolkit resource reconciliation.,0,fluxcd,gotk reconcile condition,
+fluxcd.gotk.reconcile.duration.seconds.bucket,count,,,,Bucket of the duration in seconds of a GitOps Toolkit resource reconciliation.,0,fluxcd,reconcile duration seconds,
+fluxcd.gotk.reconcile.duration.seconds.count,count,,second,,Count of the duration in seconds of a GitOps Toolkit resource reconciliation.,0,fluxcd,reconcile duration seconds,
```

Comment:
Are you sure these `count` are in seconds? 🤔

Reply:
Indeed they aren't. Neither are `.bucket` metrics. Updated, thanks for the catch!